android/app/src/main/kotlin/com/bitmark/autonomy_flutter/MainActivity.kt

```diff
@@ -34,6 +36,14 @@ class MainActivity : FlutterFragmentActivity() {
 
     var flutterSharedPreferences: SharedPreferences? = null
 
+    private fun settingFlutterView() {
+        val flutterView = io.flutter.embedding.android.FlutterView(this)
+//        val contentView = findViewById<View>(android.R.id.content).rootView
```

Comment:
can you import `io.flutter.embedding.android.FlutterView` and use `val flutterView = FlutterView(this)` only to be cleaner.

btw, please remove the unused code.